packages/replay/MIGRATION.md

```diff
@@ -0,0 +1,51 @@
+# Upgrading Replay from 0.6.x to 7.24.0
```

Comment:
l: Should we add a sentence about the version jump? Added a suggestion but feel free to disregard/change

```suggestion
# Upgrading Replay from 0.6.x to 7.24.0

The Sentry Replay integration was moved to the Sentry JavaScript SDK monorepo. Hence we're jumping from version 0.x to the monorepo's 7.x version which is shared across all JS SDK packages.  
```